react/features/video-quality/subscriber.js

```diff
@@ -238,6 +238,8 @@ function _updateReceiverVideoConstraints({ getState }) {
     let receiverConstraints;
 
     if (sourceNameSignaling) {
+        const remoteScreenSharesSourceNames = getRemoteScreenSharesSourceNames(state, remoteScreenShares);
```

Comment:
prior to this PR signaling and receiving multiple streams were coupled together. 

After this PR:
- w/ signaling w/o multi-stream, `remoteScreenShares` stores endpoint ids
- w/signaling w/ multi-stream, `remoteScreenShares` stores the fake SS participant ids which are track source names